modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala

```diff
@@ -112,7 +112,10 @@ trait VoTableParser {
 
     def parseName: ValidatedNec[CatalogProblem, NonEmptyString] =
       Validated
-        .fromOption(entries.get(adapter.nameField).flatMap(refineV[NonEmpty](_).toOption),
+        .fromOption(entries
+                      .get(adapter.nameField)
+                      .map(_.stripPrefix("NAME "))
```

Comment:
The `adapter` is used because each votable implementation of the "Standard" is different.
I think this function should be in the adapter for `simbad` and would be the identity function for other adapters.

Reply:
Agreed 👍  will make the change